src/lib/prisma-session-store.ts

```diff
@@ -214,7 +214,12 @@ export class PrismaSessionStore<M extends string = 'session'> extends Store {
       if (Array.isArray(sid)) {
         await Promise.all(sid.map(async (id) => this.destroy(id, callback)));
       } else {
-        await this.prisma[this.sessionModelName].delete({ where: { sid } });
+        // const session = await this.prisma[this.sessionModelName].findUnique({ where: { sid } });
```

Comment:
Would you mind removing the commented-out code,
and adding a comment to clarify that `deleteMany()` is being used to address a bug (with a link to the bug it is addressing) - just to make it clear for people reading this in the future why the "many" version is being used?